guides/source/active_record_validations.md

```diff
@@ -365,6 +365,36 @@ end
 
 The default error message for this helper is _"doesn't match confirmation"_.
 
+### `comparison`
+
+This check will validate a comparison between any two comparable values.
+If an option is not passed, all validations pass. It accepts a value, proc, or symbol.
```

Comment:
`It accepts a value, proc, or symbol` what is `It` referring here?

Reply:
I've updated the text to say Each option. I'm open to clearer alternatives.